wandb/fastai/__init__.py

```diff
@@ -43,23 +45,27 @@ class WandbCallback(TrackerCallback):
 
     def __init__(self,
                  learn,
-                 log=None,
-                 show_results=False,
-                 save_model=False,
+                 log="all",
+                 save_model=True,
                  monitor='val_loss',
-                 mode='auto'):
+                 mode='auto',
+                 data_type=None,
```

Comment:
We're about to change the interface of our keras callback to accept input_type and output_type.  These can often be inferred but I suggest for now you just change this to input_type.  2 different types allows us to explicitly do image classification, segmentation, etc.

Reply:
Ok, let me know once it's revised so I try to replicate it. In fastai we can often get type of data from the name of the dataset class (does not work with custom classes).